great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -38,62 +36,9 @@ class ErrorPayload(TypedDict):
     errors: List[ErrorDetail]
 
 
-class PayloadDataField(TypedDict):
-    attributes: dict
-    id: str
-    type: str
-
-
-class ResponsePayload(TypedDict):
-    data: PayloadDataField | list[PayloadDataField]
-
-
-AnyPayload = Union[ResponsePayload, ErrorPayload]
-
-
-class RequestPayloadDataField(TypedDict):
-    attributes: dict
-    id: NotRequired[str]
-    type: str
-
-
-class RequestPayload(TypedDict):
-    data: RequestPayloadDataField
-
-
-def construct_url(
-    base_url: str,
-    organization_id: str,
-    resource_name: str,
-    id: Optional[str] = None,
-) -> str:
-    url = urljoin(
-        base_url,
-        f"organizations/{organization_id}/{hyphen(resource_name)}",
-    )
-    if id:
-        url = f"{url}/{id}"
-    return url
-
-
-def construct_json_payload(
-    resource_type: str,
-    organization_id: str,
-    attributes_key: str,
-    attributes_value: Any,
-    **kwargs: dict,
-) -> RequestPayload:
-    data: RequestPayload = {
-        "data": {
-            "type": resource_type,
-            "attributes": {
-                "organization_id": organization_id,
-                attributes_key: attributes_value,
-                **kwargs,
-            },
-        }
-    }
-    return data
+class EndpointVersion(str, Enum):
+    V0 = "V0"
+    V1 = "V1"
```

Comment:
Why does the client need to care about versions other than V1?

Reply:
in order to support incremental migration - this store needs to know whether it should construct a V1 shape or a V0 shape.

Reply:
@joshua-stauffer 
Is there a doc or something that explains why we want to support this?
It seems like an unnecessary complexity.

Reply:
I see, so this doesn't necessarily mean we will release with both code paths on the OSS side.